Sources/Quick/Configuration/Configuration.swift

```diff
@@ -103,10 +109,16 @@ final public class Configuration: NSObject {
         is provided with metadata on the example that the closure is being
         run after.
     */
+#if !os(Linux)
```

Comment:
That's a great insight, yeah.

I've been working on GitHub for long enough to know the drill, but still appreciate you calling it out like that :+1: